spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -185,232 +102,212 @@
+        expect(result_context_stages_state_id[:success]).to eq(true)
+
+        expect(aamva_stub).to_not have_been_requested
       end
+    end
+
+    context 'with threatmetrix disabled' do
+      let(:proofing_device_profiling) { :disabled }
+
+      it 'does not make a request to threatmetrix' do
+        stub_vendor_requests => { threatmetrix_stub: }
 
-      context 'does call state id with an unsuccessful response from the proofer' do
-        it 'posts back to the callback url' do
-          expect(resolution_proofer).to receive(:proof).
-            and_return(Proofing::ResolutionResult.new(exception: 'error'))
-          expect(state_id_proofer).to receive(:proof).
-            and_return(Proofing::StateIdResult.new)
+        perform
 
-          perform
-        end
+        result = document_capture_session.load_proofing_result[:result]
+        result_context = result[:context]
+        result_context_stages = result_context[:stages]
+        result_context_stages_threatmetrix = result_context_stages[:threatmetrix]
+
+        expect(result[:success]).to be true
+        expect(result[:exception]).to be_nil
+        expect(result[:timed_out]).to be false
+
+        # result[:context][:stages][:threatmetrix]
+        expect(result_context_stages_threatmetrix).to be_nil
+
+        expect(threatmetrix_stub).to_not have_been_requested
       end
+    end
+
+    context 'without a threatmetrix session ID' do
+      let(:threatmetrix_session_id) { nil }
+
+      it 'does not make a request to threatmetrix' do
+        stub_vendor_requests => { threatmetrix_stub: }
 
-      context 'no state_id proof' do
-        let(:should_proof_state_id) { false }
+        perform
 
-        it 'does not call state_id proof if resolution proof is successful' do
-          expect(resolution_proofer).to receive(:proof).
-            and_return(Proofing::ResolutionResult.new)
+        result = document_capture_session.load_proofing_result[:result]
+        result_context = result[:context]
+        result_context_stages = result_context[:stages]
+        result_context_stages_threatmetrix = result_context_stages[:threatmetrix]
+
+        expect(result[:success]).to be true
+        expect(result[:exception]).to be_nil
+        expect(result[:timed_out]).to be false
+
+        # result[:context][:stages][:threatmetrix]
+        expect(result_context_stages_threatmetrix).to be_nil
 
-          expect(state_id_proofer).not_to receive(:proof)
-          perform
-        end
+        expect(threatmetrix_stub).to_not have_been_requested
       end
     end
 
     context 'a stale job' do
-      before { instance.enqueued_at = 10.minutes.ago }
-
       it 'bails and does not do any proofing' do
-        expect(instance).to_not receive(:resolution_proofer)
-        expect(instance).to_not receive(:state_id_proofer)
+        stub_vendor_requests => { aamva_stub:, instant_verify_stub:, threatmetrix_stub: }
```

Comment:
I'm not familiar with this syntax? Is this the same as:

```suggestion
        stub_vendor_requests(aamva_stub:, instant_verify_stub:, threatmetrix_stub:)
```

Reply:
If so, could we change this to the parens format? The hashrocket looks really off to me, we tend to avoid that style a lot

Reply:
It is sugar for this:

```ruby
aamva_stub, instant_verify_stub, threatmetrix_stub = stub_vendor_requests.values_at(:aamva_stub, :instant_verify_stub, :threatmetrix_stub)
```

Reply:
oh goodness I know hash destructuring shipped like that. I'm not in love with it but if you feel strongly, let's keep it

Another idea would be to assign those to @-vars inside of `stub_vendor_requests`

```ruby
stub_vendor_requests

@aamva_stub.check_something
```

WDYT?

Reply:
Hmm, I hadn't considered the ivars. That is a little indirection but I don't love the rightward assignment either. Was just trying to keep grabbing the stubs from taking up too much room.

I'll swap in ivars reaaal quick

Reply:
Done in f49ea5c46eb17a7ff7bec0d37ba3ac2c3d045fa5